.github/workflows/automation-housekeeping.yml

```diff
@@ -0,0 +1,67 @@
+name: Housekeeping
+
+on:
+  workflow_call:
+    inputs:
+      delete_head_branch:
```

Comment:
Is there a reason why these good practices are default off?

Reply:
Yes, if a repo wants these standard practices it should be clearly specified. If they were 'on' by default, omitting the variable in a repo would force the change. I feel that is the safest way to make sure unwanted changes aren't applied